test/integ/test_cursor.py

```diff
@@ -1235,3 +1236,15 @@ def test_describe(conn_cnx):
             assert constants.FIELD_ID_TO_NAME[column_types[2]] == "TEXT"
             assert "TIMESTAMP" in constants.FIELD_ID_TO_NAME[column_types[3]]
             assert len(cur.fetchall()) == 0
+
+            # test insert
+            cur.execute(
+                "create table {name} (aa int)".format(name=db_parameters["name"])
+            )
+            description = cur.describe(
+                "insert into {name}(aa) values({value})".format(
+                    name=db_parameters["name"], value="1234"
+                )
+            )
+            assert description[0][0].startswith("number of rows inserted")
```

Comment:
What should be the value in (rest of) the message? 0 or -1?

Reply:
It only has "number of rows inserted" as a part of metadata and no value after this. I  changed it to use `==`.